rpadutils/rpadutils.py

```diff
@@ -664,3 +664,24 @@ def wrapper(*args, **kwargs):
         return wraps(func)(wrapper)
 
     return decorator
+
+
+async def confirm_message(ctx, text, yemoji = "✅", nemoji = "❌", timeout = 10):
+    msg = await ctx.send(text)
+    await msg.add_reaction(yemoji)
+    await msg.add_reaction(nemoji)
+    def check(reaction, user):
+        return str(reaction.emoji) in [yemoji, nemoji] \
+               and user.id == ctx.author.id \
+               and reaction.message.id == msg.id
```

Comment:
Just as a general point, line continuations are considered bad in python. If you need to span something over multiple lines like this, either extract to named local vars (sometimes good because 'code as comments') or use parens to group.